catalog/project/kcc-namespace/validation.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: fn.kpt.dev/v1alpha1
+kind: StarlarkRun
+metadata:
+  name: validate-project-ns
+source: |
+  def get_tenant_ns(resource_list):
+    for resource in resource_list["items"]:
+      if resource["kind"] == "Namespace":
+        return resource["metadata"]["name"]
+    fail("unable to find tenant project namespace")
+
+  def get_projects_ns(resource_list):
```

Comment:
We should probably instead look for where the service account is created.

Reply:
The SA is created in the management ns which could be different from the projects ns. We could assert that tenant ns is not equal to management, project and network ns, current we only do projects ns.

Reply:
Let's check that it's not equal to management or project namespace. Thanks!

Reply:
Added